Testreglar/2.4.4/Nett/nett-2.4.4a.json

```diff
@@ -0,0 +1,232 @@
+{
+	"namn": "Nett-2.4.4a Formål med lenker går fram av lenketeksten, eller lenketeksten og konteksten saman 2023",
+	"id": "nett-2.4.4a",
+	"testlabId": 473,
+	"versjon": "1.0",
+	"type": "Nett",
+	"spraak": "nn",
+	"kravTilSamsvar": "<p>Lenker som inneholder tekst oppfyller kravet dersom:</p>\r\n<ul>\r\n<li>Lenketeksten alene beskriver formålet med lenken eller</li>\r\n<li>Lenketeksten sammen med programmeringsmessig bestemt lenkekontekst beskriver formålet med lenken. For programmatisk bestemt lenkekontekst vurderes tekst som står i\r\n<ul>\r\n<li>samme avsnitt, liste eller celle i en tabell som lenken gjør eller tilsvarende eller</li>\r\n<li>en tabelloverskrift som er knyttet til tabellcellen som lenken står i eller tilsvarende</li>\r\n</ul>\r\n</li>\r\n</ul>",
```

Comment:
Må oversettes til nynorsk. Heller "For lenker som inneheld tekst gjeld følgande: (…)"

Reply:
Hvorfor skal krav til samsvar oversettes? Etter oppdatert tolkning er samtlige krav til samsvar på bokmål. Det gjelder for alle testregler.

Reply:
Fordi det er en del av testreglen. Vi har vell mange testreglar på nynorsk, der krav til samsvar er på nynorsk?